libcontainer/container_linux.go

```diff
@@ -1504,11 +1504,14 @@ func (c *linuxContainer) criuSwrk(process *Process, req *criurpc.CriuReq, opts *
 	// cmd.Process will be replaced by a restored init.
 	criuProcess := cmd.Process
 
+	var st *os.ProcessState
 	defer func() {
-		criuClientCon.Close()
-		_, err := criuProcess.Wait()
-		if err != nil {
-			return
+		if st == nil {
+			criuClientCon.Close()
+			_, err := criuProcess.Wait()
+			if err != nil {
+				logrus.Warningf("wait on criuProcess returned %v", err)
```

Comment:
nit: `s/Warningf/Warnf/` (both are OK but looks like `Warnf` is used everywhere else in runc code).

Reply:
Addressed.